src/MainWindow/main_window.cpp

```diff
@@ -160,6 +160,18 @@ void MainWindow::CloseOpenedTabs() {
   }
 }
 
+void MainWindow::closeEvent(QCloseEvent* event) {
+  QMessageBox::StandardButton closeBtn = QMessageBox::warning(
+      this, "Warning", tr("Do you really want to close the project?\n"),
+      QMessageBox::Cancel | QMessageBox::Ok);
```

Comment:
Why did you change to "Ok"? In this context "Yes" suites better.

Reply:
@volodymyrkochyn , Okay and Then Should I also change "cancel" to "No"?

Reply:
Yes, that will be better.

Reply:
Fixed.

Reply:
and update screenshot